Cargo.toml

```diff
@@ -24,4 +25,5 @@ tempfile = "3.0.7"
 atomicwrites = "0.2.3"
 vec1 = "1.1.0"
 proptest = "0.9.1"
-nix = "0.14.0"
\ No newline at end of file
+nix = "0.14.0"
+human-panic = { git = "https://github.com/rust-cli/human-panic.git", rev = "70db9489" }
```

Comment:
This could just be set to master and then enforced by the lockfile, am unsure what is preferred.

I should probably add a comment as well.